licenses/admin.py

```diff
@@ -1,6 +1,25 @@
 from django.contrib import admin
 
-from licenses.models import LegalCode, License
+from licenses.models import LegalCode, License, TranslationBranch
+
+
+@admin.register(TranslationBranch)
+class TranslationBranchAdmin(admin.ModelAdmin):
+    list_display = [
+        "branch_name",
+        "version",
+        "language_code",
+        "last_transifex_update",
+        "complete",
```

Comment:
Should we include a url to the staging/preview site for the translation branch here?

Reply:
I think the preview site URLs will be automatic based on the translation branch name. We don't yet know what they'll be, but there shouldn't be any need to store the individual translation branch preview URLs.

I will add a preview site link to the pages, though, with a note that it doesn't work yet or something.